Doc/library/ipaddress.rst

```diff
@@ -40,7 +40,9 @@ IP addresses, networks and interfaces:
 .. function:: ip_address(address)
 
    Return an :class:`IPv4Address` or :class:`IPv6Address` object depending on
-   the IP address passed as argument.  Either IPv4 or IPv6 addresses may be
+   the IP address passed as argument.
+   *address* is a string or integer or bytes representing the IP address.
+   Either IPv4 or IPv6 addresses may be
    supplied; integers less than ``2**32`` will be considered to be IPv4 by default.
    A :exc:`ValueError` is raised if *address* does not represent a valid IPv4
    or IPv6 address.
```

Comment:
It makes the diff ugly, but maybe it's still worth reflowing these to fit the style in the rest of the file? Is there a convention when making these changes? (would love a resource if you have one)

```
   Return an :class:`IPv4Address` or :class:`IPv6Address` object depending on
   the IP address passed as argument. *address* is a string or integer or bytes
   representing the IP address. Either IPv4 or IPv6 addresses may be supplied;
   integers less than ``2**32`` will be considered to be IPv4 by default. A
   :exc:`ValueError` is raised if *address* does not represent a valid IPv4 or
   IPv6 address.
```

Reply:
I don't have an opinion on that, I will do anything that is considered the best by maintainers 🙂

Reply:
The feedback I got from [this PR](https://github.com/python/cpython/pull/29633#discussion_r753558562) was that it's generally preferred to keep the diff on docs PRs as small as possible, even if that means that some lines go way over (or way under) the 80-char PEP8 limit.

Reply:
If a line really goes way over 80 characters, then split it in two 🙂